docs/docs/guide/v2-migration.md

```diff
@@ -22,9 +22,6 @@ If you previously used `dbt` or `dbt-<adapter>` ([available adapters documentati
 This is easy to do! Following the instructions from above to discover and install your chosen adapter:
 
 ```bash
-# list available utilities plugins
-meltano discover utilities
```

Comment:
Will these changes cause problems with the v3 pr? cc @WillDaSilva

Reply:
I don't believe so.